### ISSUE TITLE
eval_model() with a Question Answering Model

### ISSUE THREAD

**Issue by mineshmathew** [ThilinaRajapakse/simpletransformers#488]
When I run `model.eval_model ()`, three files are written to the output folder, with probabilities for each prediction, n best predictions etc. 

But how do i get an evaluation score?

I suppose F1 score is the standard evaluation score, is there anyway I can get the evaluation score from eval_model() call.

Also what is the  _null_odds_test.json_ file created in the outputs folder after evaluation ?

**Comment by mineshmathew** [comment]
I have one more issue - even though I have two GPUs and set n_gpus:2  (`CUDA_VISIBLE_DEVICES` shows _0.1_ , t) my evaluation script for Question answering uses only a single GPU.  Should I set some data_parallel = true style flag somewhere?

Also what is your suggestion on batch_size i should use for squad style data. Even when I used 256 batch size for evaluation the GPU memory wasn't even half full.
